packages/loading-skeleton/src/LoadingSkeleton/components/LoadingGraphic/LoadingGraphic.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { HTMLAttributes } from "react"
+import classnames from "classnames"
+import { OverrideClassName } from "@kaizen/component-base"
+import styles from "../styles.scss"
+
+export type GraphicSizes =
+  | "small"
+  | "medium"
+  | "large"
+  | "xlarge"
+  | "xxlarge"
+  | "scene"
+
+const graphicSizesRem = new Map<string, number>([
```

Comment:
Just a question really: Could `<string, number>` here be `<GraphicSizes, number>`? It is one of those, isn't it?